SampleApp/SampleApp/Screens/AudienceCheckingView.swift

```diff
@@ -39,8 +44,11 @@ struct AudienceCheckingView: View {
             message: result)
     }
 
-    private func checkAudience() {
-        RealifeTech.Audiences.deviceIsMemberOfAudience(audienceId: audienceId) { response in
+    private func checkAudience() throws {
```

Comment:
Any reason we can't use ? syntax here?

Reply:
It's for error handling to show an alert when device hasn't registered, otherwise nothing will happen on the app.
Or do you think I should remove it, in order to keep the sample app as simple as possible?